web/cypress/e2e/app.cy.ts

```diff
@@ -0,0 +1,16 @@
+import "cypress-axe";
+
+describe("ARC Portal UI", () => {
+  it("shows the ARC portal phrase on the homepage", () => {
+    cy.visit("/");
+
+    cy.contains("ARC portal").should("be.visible");
+    cy.contains("fibble").should("not.exist");
+
+    // run accessibility check
+    cy.injectAxe();
```

Comment:
should be a separate test(?)

Reply:
I've now extracted it into a separate test. The exact testing approach is yet to be decided though. For instance, shall we have a separate testing file dedicated to running a full accessibility testing suite?